packages/design-picker/src/components/index.tsx

```diff
@@ -40,11 +41,12 @@ const DesignPicker: React.FC< Props > = ( {
 	),
 	premiumBadge,
 	isGridMinimal,
+	theme = 'light',
 } ) => {
 	const { __ } = useI18n();
 
 	return (
-		<div className="design-picker">
+		<div className={ classnames( 'design-picker', `theme-${ theme }` ) }>
```

Comment:
Don't have a strong opinion on className conventions, but it seems like we're using some sort of BEM for design picker

What about 
```suggestion
		<div className={ classnames( 'design-picker', `design-picker--theme-${ theme }` ) }>
```